File: Poin B.php/Disk.php
<?php
class Disk
{
    //atribute
    private $type = "";
    private $capacity = "";
    private $price = 0;

    //constructor
    function __construct($type = "", $capacity = "", $price = 0)
    {
        // konstruktor
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }
    //get set methods
    function setType($type)
    {
        $this->type = $type;
    }
    function getType()
    {
        return $this->type;
    }
    function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }
    function getCapacity()
    {
        return $this->capacity;
    }
    function setPrice($price)
    {
        $this->price = $price;
    }
    function getPrice()
    {
        return $this->price;
    }
    //output methods
    function output()
    {
        echo "Disk                      : " . $this->getType() ." ".$this->getCapacity(). "<br>";
        echo "Harga                     : " . $this->getPrice() . "<br>";
    }


    //destructor
    function __destruct()
    {
    }
}

File: Poin B.java/Disk.java
public class Disk {
    //atribut
    private String type;
    private String capacity;
    private int price;

    //konstruktor
    public Disk(String type, String capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }
    public Disk() {
    }


    //get set methods
    public int getPrice() {
        return price;
    }
    public String getType() {
        return type;
    }
    public void setType(String type) {
        this.type = type;
    }
    public String getCapacity() {
        return capacity;
    }
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    //output
    public void output(){
        System.out.println("Disk                    : " + this.getType() + " " + this.getCapacity());
        System.out.println("Price                   : " + this.getPrice());
    }

    
    
}
